### PR TITLE
[TASK-24] add profiles configuration

### PR DESCRIPTION
**Pull request by kateryna-mykh**

### CODE REVIEW

File: saas/mic-hex-eims/boot/src/main/resources/application.properties [hunk 1]
```diff
@@ -6,3 +6,5 @@ spring.application.name=${application.banner.code}
 spring.webflux.base-path=${application.banner.code}
 
 server.port=8000
+
+spring.profiles.active=default
```
Comment (sokimaaa) [inline saas/mic-hex-eims/boot/src/main/resources/application.properties]: No need to have this property in the our repo. 
The same for others.
```suggestion
```

File: saas/mic-hex-eims/boot/src/main/resources/application-cloud.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+discovery.enabled=true
```
Comment (sokimaaa) [inline saas/mic-hex-eims/boot/src/main/resources/application-cloud.properties]: I am not sure this property even exists? 
Probably a typo. 
```suggestion
eureka.client.enabled=true
```

File: library/lib-discovery-client-starter/src/main/java/com/sokima/lib/discoveryclient/starter/config/DiscoveryClientAutoConfiguration.java [hunk 3]
```diff
@@ -1,11 +1,14 @@
 package com.sokima.lib.discoveryclient.starter.config;
 
-import org.springframework.boot.autoconfigure.AutoConfiguration;
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 import org.springframework.context.annotation.PropertySource;
 
+@Configuration
```
Comment (sokimaaa) [inline library/lib-discovery-client-starter/src/main/java/com/sokima/lib/discoveryclient/starter/config/DiscoveryClientAutoConfiguration.java]: Can we leave `@Autoconfiguration`?
```suggestion
@Configuration
```

File: saas/mic-hex-eims/boot/src/main/resources/application-cloud.properties (hunk 2 above)
Comment (kateryna-mykh) [reply saas/mic-hex-eims/boot/src/main/resources/application-cloud.properties]: @sokimaaa, I also was puzzled, but set this property, according to your answer in messenger). I will fix this.

File: library/lib-discovery-client-starter/src/main/java/com/sokima/lib/discoveryclient/starter/config/DiscoveryClientAutoConfiguration.java (hunk 3 above)
Comment (kateryna-mykh) [reply library/lib-discovery-client-starter/src/main/java/com/sokima/lib/discoveryclient/starter/config/DiscoveryClientAutoConfiguration.java]: We can.

**Pull request merged**